passes/cmds/setundef.cc

```diff
@@ -243,7 +243,7 @@ struct SetundefPass : public Pass {
 			{
 				for (auto *cell : module->selected_cells()) {
 					for (auto &parameter : cell->parameters) {
-						for (auto bit : parameter.second) {
+						for (auto &bit : parameter.second.bits()) {
```

Comment:
This is what the line was prior to #4516 
@widlarizer can you check this?

Reply:
Thanks for the ping, it's a case just like #4951 where I broke something and didn't notice partly due to missing test coverage